integration-tests/deployment/cs_test.go

```diff
@@ -84,6 +86,7 @@ func TestDeploy(t *testing.T) {
 		GasPrice: big.NewInt(1e17),
 		TokenPrices: map[string]*big.Int{
 			tvm.TonTokenAddr.String(): big.NewInt(99),
+			linkAddr.String():         big.NewInt(20),
```

Comment:
@nicolasgnr we also need to configure the LINK token price in deployments, something that was missing in tests before.

The deployment sequence also currently doesn't deploy a token